crates/driver/src/domain/competition/order/mod.rs

```diff
@@ -148,6 +148,7 @@ impl Order {
         } else {
             self.sell
         }
+        // todo not sure about this one
```

Comment:
@cowfee if this is what is sent to solver, it should probably be 
```
pub fn solver_sell(&self) -> eth::Asset {
   self.sell
}
```

since now `surplus_fee` is always 0.

Reply:
You should be able to remove the `solver_sell` function and just use `sell` (since we no longer need to "massage" the sell amount that we send to the solver for FoK limit orders.

Reply:
Removed.